modules/virt-creating-vm-instancetype.adoc

```diff
@@ -45,6 +45,11 @@ endif::[]
 . In the web console, navigate to *Virtualization* -> *Catalog*.
 +
 The *InstanceTypes* tab opens by default.
++
+[NOTE]
+====
+* When configuring a downward metrics device on an {ibm-z-name} system using a VM preference, the `spec.preference.name` value must be set to `rhel.9.s390x` or another available preference with the format `*.s390x`.
```

Comment:
Just a suggestion in case you wish to consider polishing this sentence:
```suggestion
* When configuring a downward-metrics device on an {ibm-z-name} system that uses a VM preference, set the `spec.preference.name` value to `rhel.9.s390x` or another available `*.s390x` preference.
```